support/ebpf/sched_monitor.ebpf.c

```diff
@@ -7,20 +7,23 @@
 #include "types.h"
 
 // See /sys/kernel/debug/tracing/events/sched/sched_process_free/format
-// for struct layout.
-struct sched_process_free_ctx {
+// for struct layout. This is pre-6.16 format which uses a fixed-size
+// (TASK_COMM_LEN) array for comm.
+struct sched_process_free_ctx_old {
```

Comment:
```suggestion
struct sched_process_free_ctx_pre616 {
```

wdyt to make it more explicit about the kernel cutoff? Or